src/Material.h

```diff
@@ -16,6 +16,9 @@
 #include "log.h"
 #endif
 
+/** The minimum allowable total cross-section value for any group */
```

Comment:
Good call - I revised the comment for ``ZERO_SIGMA_T`` in "Material.h". Let me know if you think it makes sense.